backend/de.metas.business/src/main/java/de/metas/costing/methods/AveragePOCostingMethodHandler.java

```diff
@@ -153,6 +153,13 @@ private CostDetailCreateResult createCostDetailAndAdjustCurrentCosts(final CostD
 		{
 			final CostPrice price = currentCosts.getCostPrice();
 			final CostAmount amt = price.multiply(qty).roundToPrecisionIfNeeded(currentCosts.getPrecision());
+			
+			if (qty.isZero())
```

Comment:
wondering if is not better to move this on the same level like the if/else.

Reply:
moved